build.gradle

```diff
@@ -72,42 +73,67 @@ dependencies {
 	testCompile 'org.mockito:mockito-all:2.0.2-beta'
 }
 
-task allPlatform(type: Zip, group: 'release', dependsOn: ':shadowJar') {
+def assetsDirectory = file("${buildDir}/assets")
+git {
+	assets {
+		url 'https://github.com/triplea-game/assets.git'
+		directory assetsDirectory
+		branch 'master'
+		singleBranch true
+	}
+}
+import com.install4j.gradle.Install4jTask
+
+task updateAssets(dependsOn: ['gitCloneAssets', 'gitPullAssets']) {
+}
+
+
+task renameShadowJar(type: Copy, group: 'release', dependsOn: [shadowJar]) {
+	ext.output = file("$project.buildDir/libs/all/triplea.jar")
+	from shadowJar.archivePath
+	into ext.output.parent
+	rename shadowJar.archivePath.name, output.name
+}
+
+task allPlatform(type: Zip, group: 'release', dependsOn: [renameShadowJar, updateAssets]) {
 	classifier 'all_platforms'
-	['assets', 'dice_servers', 'doc', 'license', 'maps', 'old'].each { folder ->
+	['assets', 'dice_servers', 'license', 'maps', 'old'].each { folder ->
 		from(folder) {
 			into(folder)
 		}
 	}
-	from('icons') {
+	from("${assetsDirectory}/doc"){
+		into('doc')
+	}
+	from("${assetsDirectory}/icons") {
 		into('icons')
-		exclude('Icon.zip')
-		exclude('icon.icns')
+		exclude("icon.icns")
 	}
 	['readme.html', 'changelog.txt', 'system.ini', 'TripleA_RuleBook.pdf',
 	 'run-headless-game-host-mac-os.sh', 'run-headless-game-host.sh', 'run-headless-game-host-windows.bat',
 	 'triplea_unix.sh', 'triplea_windows.bat', 'triplea_mac_os_x.sh'].each { fileName ->
 		from(fileName)
 	}
-	from("build/libs/triplea-${project.version}-all.jar") {
+	from(renameShadowJar.output) {
 		into('bin')
-		rename "triplea-${project.version}-all.jar", 'triplea.jar'
 	}
 }
 
-task bots(type: Zip, group: 'release', dependsOn: ':shadowJar') {
+task bots(type: Zip, group: 'release', dependsOn: [renameShadowJar, updateAssets]) {
 	classifier 'bots'
 	from('assets') {
 		exclude('resources')
 		exclude('sounds')
 		exclude('territoryEffects')
 	}
-	from('icons') {
+	from("${assetsDirectory}/icons") {
 		into('icons')
-		exclude('Icon.zip')
-		exclude('icon.icns')
+		exclude("icon.icns")
 	}
-	['dice_servers', 'doc', 'icons', 'license'].each { folder ->
+	from("${assetsDirectory}/doc"){
+		into('doc')
+	}
+	['dice_servers', 'license'].each { folder ->
```

Comment:
Why do bot's need a dice servers config? The dice server config only needs to live on the dice servers, is it needed elsewhere?